site/content/en/v1.4/tasks/operations/gateway-namespace-mode.md

```diff
@@ -16,8 +16,12 @@ In standard deployment mode, Envoy Gateway creates all data plane resources in t
 
 Gateway Namespace Mode changes this behavior by placing Envoy Proxy data plane resources like Deployments, Services and ServiceAccounts in each Gateway's namespace, providing stronger isolation and multi-tenancy.
 
-Traditional deployment mode uses mTLS where both the client and server authenticate each other. However, in Gateway Namespace Mode, we've shifted to server-side TLS and JWT token validation between infra and control-plane.
```

Comment:
can we add a line in the beginning around motivation, explaining why the default is Controller Namespace mode (to operate with minimum permissions) and why someone would choose gateway namespace mode (isolation of Envoy proxy fleet across tenants)